Decred Wallet.xcodeproj/project.pbxproj

```diff
@@ -1007,6 +1071,288 @@
 /* End PBXVariantGroup section */
 
 /* Begin XCBuildConfiguration section */
+		25F34E9923080A8200A999B2 /* Mainnet Debug */ = {
```

Comment:
See my previous comment. These changes should go.

Reply:
- should be resolved in 07f7e91